src/components/layouts/EmptyState/DashboardEmptyState.tsx

```diff
@@ -44,7 +46,7 @@ const DashboardEmptyState = ({ userType }: { userType: UserType }) => {
                   : 'Tasks will show here once they’ve been assigned to you. '}
               </Typography>
             </Stack>
-            {userType == UserType.INTERNAL_USER && (
+            {userType === UserType.INTERNAL_USER && !!!pathname.includes('client') && (
```

Comment:
Shouldn't a `!pathname.includes('client')` be enough here?

Reply:
It was just a personal preference though single `!` operator is also enough :D

Reply:
I think the main question here is why `userType === UserType.INTERNAL_USER` isn't working @aatbip 

This fix just seems like patchwork over another deeper problem